pkg/clusteragent/admission/mutate/auto_instrumentation.go

```diff
@@ -290,12 +291,14 @@ func injectAutoInstruConfig(pod *corev1.Pod, libsToInject []libInfo) error {
 				}})
 		default:
 			metrics.LibInjectionErrors.Inc(langStr)
+			metrics.MutationErrors.Inc(metrics.LibInjectionMutationType, "unsupported language", langStr)
 			lastError = fmt.Errorf("language %q is not supported. Supported languages are %v", lib.lang, supportedLanguages)
 			continue
 		}
 
 		if err != nil {
 			metrics.LibInjectionErrors.Inc(langStr)
+			metrics.MutationErrors.Inc(metrics.LibInjectionMutationType, "", langStr)
```

Comment:
Maybe we would need a specific `reason` tag for this edge case.

Reply:
good point! thoughts on something like `error setting auto-injection configuration` ?

Reply:
It might be too long, since these tags will be used in the UI, I think that the shorter they are, the easier we can use them. I suggest something like `failed_injection`

Also I am not really familiar with this part of the code but I wonder if it might be worth logging the error somewhere. It looks like we only report the last error.

Reply:
that makes sense! how about `requirements config error`? since it's already tagged with the `LibInjectionMutationType`